config_defaults.yml

```diff
@@ -558,18 +558,13 @@ Gemfile:
     ruby_versions:
       '2.1.9':
         checks:
-          - 'lint validate metadata_lint'
           - spec
         puppet_version: '~> 4.0'
-      '2.4.1':
-        checks:
-          - 'lint validate metadata_lint'
-          - rubocop
-          - spec
-        puppet_version: '~> 5.0'
       '2.4.4':
         checks:
           - 'lint validate metadata_lint'
```

Comment:
Just noticed this. Probably don't need validate and lint in the metadata_lint task if you're using `sytnax` and `lint` in another job

Reply:
Honestly I think rubocop, syntax, lint, validate, metadata_lint should all happen at the same time.  I'll don't see any reason to incur the cost of a new container to run them separately.  

What do you think?

Reply:
I think it's good to keep `rubocop` separate for now since that's how I've seen it in most repos and internally. The `validate` task can be removed as I believe the `syntax` task covers that.